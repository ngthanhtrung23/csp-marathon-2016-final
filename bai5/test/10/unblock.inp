1.22..
133345
00.645
77.689
.A.689
.ABBB.
