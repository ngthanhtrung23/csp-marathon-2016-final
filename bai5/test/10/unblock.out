16
0 R 1
2 L 1
4 U 1
5 U 1
7 R 1
1 D 2
3 L 1
6 U 2
7 R 1
9 D 1
A U 1
B L 2
8 D 1
7 R 2
6 D 3
0 R 3

